src/_components/AccessControl/Authorize.jsx

```diff
@@ -19,7 +23,7 @@ export default class Authorize extends React.Component {
     return (
       <div className="app background">
         <ArtModal
-          open
+          modalOpen={this.state.modalOpen}
```

Comment:
As I'm not seeing anywhere that will require state being update (`handleToggleModal` is not being used anywhere), why don't you just declare a variable and use that? Something like:
```js
const open = true;

return (
  <div className="app background">
    < ArtModal
      modalOpen={open}
      ...
    />
  </div>
)

Reply:
Im the one who forgot to pass `handleToggleModal`. But it has to be passed either way.